api/V1/recordingUtil.js

```diff
@@ -187,8 +187,72 @@ async function addTag(request, response) {
   });
 }
 
+// reprocessAll expects request.body.recordings to be a list of recording_ids
+// will mark each recording to be reprocessed
+async function reprocessAll(request, response) {
+  recordings =  request.body.recordings
+  var responseMessage = {
+    statusCode: 200,
+    messages: [],
+  }
+  for (var i = 0; i < recordings.length; i++){
+    resp = await reprocessRecording(request.user, recordings[i])
+    if(resp.statusCode != 200){
+      responseMessage.statusCode = resp.statusCode
+      responseMessage.push(resp.messages[0])
+    }
+  }
+
+  responseUtil.send(response, responseMessage);
+  return
+}
+
+// reprocessRecording marks supplied recording_id for reprocessing, 
+// under supplied user privilages
+async function reprocessRecording(user,recording_id){
```

Comment:
This guts of this should probably be a method on a Recording model object. Then we can retrieve the recording and call `rec.reprocess(user)` .  Of course, that shouldn't be returning HTTP status codes.